app/controllers/v0/sessions_controller.rb

```diff
@@ -33,6 +33,9 @@ def saml_callback
       if @saml_response.is_valid? && persist_session_and_user
         async_create_evss_account(@current_user)
         redirect_to SAML_CONFIG['relay'] + '?token=' + @session.token
+
+        obscure_token = Session.obscure_token(@session.token)
+        Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```

Comment:
It borders on logging sensitive PII to the logs.  The `@session.uuid` is permanent per user and is the primary key ID.me uses for their users.  If an attacker were to breach another SP to get the mapping of uuid--->user PII or a rogue ID.me employee got a hold of our logs, they could map uuid to a real life person.  Perhaps we could hash the uuid instead of logging it directly?  That way, we can still track each user, but there would be no way someone could read our logs and track it back to a specific ID.me user.

Also, you are correct about the lack of redundancy - my mistake.